.gitignore

```diff
@@ -1,3 +1,10 @@
 # Simple stuff modules should ignore
 *.iml
-build.gradle
\ No newline at end of file
+build.gradle
+
+# Eclipse file
+.project
+.classpath
+.settings/
+bin/
+build/
```

Comment:
Tiny suggestion: add a newline at the bottom of all text files (some best practice that GitHub encourages with these little red marks)